From 1d8ee0acaadc0536390657cabf969f181a8817a7 Mon Sep 17 00:00:00 2001
From: Emerson Pinter <dev@pinter.com.br>
Date: Wed, 11 Jul 2012 23:44:24 -0300
Subject: [PATCH] graphics: Add directives for compatibility with older EGL
 blobs (1/2)

Port from CM9.
Original author Ricardo Cerqueira (commit
fc496c0607d0d2863ff9587019e890ab806ea49d)

Change-Id: I2789a98a7069d90da30ec1231882b697f2140fce
---
 .../colorconversion/SoftwareRenderer.cpp           |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/media/libstagefright/colorconversion/SoftwareRenderer.cpp b/media/libstagefright/colorconversion/SoftwareRenderer.cpp
index 8673bad..201a0ed 100644
--- a/media/libstagefright/colorconversion/SoftwareRenderer.cpp
+++ b/media/libstagefright/colorconversion/SoftwareRenderer.cpp
@@ -65,6 +65,7 @@ SoftwareRenderer::SoftwareRenderer(
     size_t bufWidth, bufHeight;
 
     switch (mColorFormat) {
+#ifndef MISSING_EGL_PIXEL_FORMAT_YV12
         case OMX_COLOR_FormatYUV420Planar:
         case OMX_TI_COLOR_FormatYUV420PackedSemiPlanar:
         {
@@ -77,6 +78,7 @@ SoftwareRenderer::SoftwareRenderer(
 
             // fall through.
         }
+#endif
 
         default:
             halFormat = HAL_PIXEL_FORMAT_RGB_565;
-- 
1.7.9.1

